modules/programs/neomutt.nix

```diff
@@ -89,6 +152,17 @@ let
     };
   };
 
+  mkNotmuchVirtualboxes = virtualMailboxes: ''
+    virtual-mailboxes \
+      ${
+        concatStringsSep "\n" (map ({ name, query, limit, type }:
+          ''
+            "${name}" "notmuch://?query=${urlencode query}${
+              optionalString (limit != null) "&limit=${toString limit}"
+            }${optionalString (type != null) "&type=${type}"}" \'')
```

Comment:
Adding the backslash after the last line does not seem to work for me. There's no error when starting up NeoMutt, but no mailboxes are present either. As `virtual-mailboxes` just appends a new mailbox to the list, could this function be
```nix
mkNotmuchVirtualboxes = virtualMailboxes: ''                                                                               
  ${                                                                                                                       
    concatStringsSep "\n" (map ({ name, query, limit, type }:                                                              
      ''                                                                                                                   
        virtual-mailboxes "${name}" "notmuch://?query=${urlencode query}${                                                 
          optionalString (limit != null) "&limit=${toString limit}"                                                        
        }${optionalString (type != null) "&type=${type}"}"'')                                                              
      virtualMailboxes)                                                                                                    
  }'';    
```
instead?